src/ui/interface.js

```diff
@@ -68,6 +68,21 @@ module.exports = function Interface(options) {
       return true;
     });
 
+    $("#sample-image__select").click(function(e){
```

Comment:
This looks good; it'll only run if there is an HTML DOM element with an id of `sample-image__select`, so this won't break v1. However, 2 thoughts - 

1. should we mark this as a v2 feature with a comment, so people know, and if we ever clean up all v1 code we can tell it apart
2. should we otherwise explain this briefly in a comment, since it sounds a little mysterious?

Reply:
What's cool is that this means ANY image could be used as a sample image, it just has to have the id `sample-image__select`. I suggest that we change that to a class, since strictly, ids should be used only once. Then someday someone may want to add, for example, a bunch of different examples in a kind of menu, they could use this. So the class could be maybe... `.sample-image`?

Reply:
But perhaps that means we should then apply the class to just the individual images, rather than the whole container. Let me make a suggestion!